content/en/docs/writing-policies/cleanup.md

```diff
@@ -66,6 +66,46 @@ rules:
   - delete
 ```
 
+### Deletion Propagation Policy
```

Comment:
This section should not be in the main cleanup policy body

Reply:
Okay.